packages/ng/file-upload/base-file-upload/base-file-upload.component.ts

```diff
@@ -0,0 +1,73 @@
+import { booleanAttribute, Component, computed, inject, input, LOCALE_ID, output } from '@angular/core';
+import { getIntl } from '@lucca-front/ng/core';
+import { LU_FILE_UPLOAD_TRANSLATIONS } from '../file-upload.translate';
+import { formatSize, MEGA_BYTE } from '../formatter';
+
+let nextId = 0;
+
+@Component({
+	selector: 'lu-base-file-upload',
+	template: '',
+})
+export class BaseFileUploadComponent {
```

Comment:
This component should not be used right? We could juste use `@Directive()` (no params) and `abstract class`